objgraph.py

```diff
@@ -1016,9 +1020,13 @@ def _show_graph(objs, edge_func, swap_source_target,
                 srcnode, tgtnode = target, source
             else:
                 srcnode, tgtnode = source, target
-            elabel = _edge_label(srcnode, tgtnode, shortnames)
-            f.write('  %s -> %s%s;\n' % (_obj_node_id(srcnode),
-                                         _obj_node_id(tgtnode), elabel))
+            for elabel, _ in itertools.zip_longest(
```

Comment:
This doesn't work on Python 2.7.

I'd have to stop and think whether I want to drop Python 2.7 compatibility at this point.